src/src/AdminBundle/Repository/AttendeeRepository.php

```diff
@@ -0,0 +1,108 @@
+        $qb = $this->_em->createQueryBuilder('a');
+        $qb->select('count(a.id)')
+            ->from('AdminBundle:Attendee', 'a');
+        return $qb->getQuery()->getSingleScalarResult();
+    }
+    
+    public function getAttendeesFilteredByEvent(
+        $searchedNameEmail,
+        $searchedEvent,
+        $searchedIsSubscribed,
+        $numItemsPerPage,
+        $page
+    ){
+        $selectMailingList = $this->_em->createQueryBuilder('r')
+                            ->select('a')
+                            ->distinct('a.id')
+                            ->from('AdminBundle:Registration', 'r');
+        return $this->getSearchByEventQuery($selectMailingList, $searchedNameEmail, $searchedEvent, $searchedIsSubscribed)
+                    ->orderBy('a.id', 'ASC')
+                    ->setFirstResult(($page - 1) * $numItemsPerPage)
+                    ->setMaxResults($numItemsPerPage)
+                    ->getQuery()
+                    ->getResult();
+    }
+
+    public function getCountOfAttendeesFilteredByEvent(
+        $searchedNameEmail,
+        $searchedEvent,
+        $searchedIsSubscribed
+    ){
+        $selectCounPages = $this->_em->createQueryBuilder('r')
+                            ->select('COUNT(DISTINCT a.id)')
+                            ->from('AdminBundle:Registration', 'r');
+        return $this->getSearchByEventQuery($selectCounPages, $searchedNameEmail, $searchedEvent, $searchedIsSubscribed)
+                    ->getQuery()
+                    ->getSingleScalarResult();
+    }
+
+    public function getAttendeesFilteredWithoutEvent(
+        $searchedNameEmail,
+        $searchedIsSubscribed,
+        $numItemsPerPage,
+        $page
+    ){
+        $selectMailingList = $this->_em->createQueryBuilder()
+                                ->select('a')
+                                ->from('AdminBundle:Attendee', 'a');
+        return $this->getSearchByNameEmailQuery($selectMailingList, $searchedNameEmail, $searchedIsSubscribed)
+                    ->orderBy('a.id', 'ASC')
+                    ->setFirstResult(($page - 1) * $numItemsPerPage)
+                    ->setMaxResults($numItemsPerPage)
+                    ->getQuery()
+                    ->getResult();
+    }
+
+    public function getCountOfAttendeesFilteredWithoutEvent(
+        $searchedNameEmail,
+        $searchedIsSubscribed
+    ){
+        $selectCountPages = $this->_em->createQueryBuilder()
+                            ->select('count(a.id)')
+                            ->from('AdminBundle:Attendee', 'a');
+        return $this->getSearchByNameEmailQuery($selectCountPages, $searchedNameEmail, $searchedIsSubscribed)
+                    ->getQuery()
+                    ->getSingleScalarResult();
+    }
+
+    private function getSearchByEventQuery($selectQuery, $nameEmail, $event, $isSubscribed) {
+        $result = $selectQuery
+            ->innerJoin('AdminBundle:Attendee', 'a', 'WITH', 'a.id = r.attendeeId')
```

Comment:
osobne preferujem `AdminBundle:Attendee:class` lebo tam potom nemas stringy a je to viac type safe podla mna